substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SystemPropertiesSupport.java

```diff
@@ -48,6 +50,11 @@
  */
 public abstract class SystemPropertiesSupport {
 
+    public static class SystemPropertyFeatureOptions{
+        @Option(help = "Report the usage of incompatible property as exception when it is not explicitly set)")//
+        public static final RuntimeOptionKey<Boolean> ReportIncompatibleSystemPropertyError = new RuntimeOptionKey<>(false);
```

Comment:
I would name it `ReportUnsupportedSystemPropertyErrors`.

Reply:
> After thinking about the name for a while, my suggestion is to call it UNDEFINED_PROPERTIES (with the appropriate changes of the option and classes, fields, ...). That name avoids all sorts of legal hassles.

As Chris suggested, I think "undefined" would be a better name.

Reply:
Correct, please use `undefined`